haminfo/mqtt/filters.py

```
@@ -0,0 +1,227 @@
+        """Create a weather report from packet data.
+
+        Args:
+            aprs_data: Packet data dictionary.
+            station: The weather station this report belongs to.
+
+        Returns:
+            The original packet if report was created, None otherwise.
+        """
+        try:
+            report = WeatherReport.from_json(aprs_data, station.id)
+        except Exception as ex:
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)